tests/privacymanager.spec.ts

```diff
@@ -0,0 +1,9 @@
+import generateCMPTests from "../playwright/runner";
+
+// FIXME: the popup closes too fast, before sending the success report
```

Comment:
Could you elaborate more here? Where's the issue here preventing us testing this?

Reply:
This rule operates inside an iframe. Initially I was confident that the iframe is removed before it can send a success report.
But now that I double-checked, it's actually because of a polyfilled Promise that doesn't support `Promise.any()`. Our test suite is not using isolated worlds at the moment, so it's not working there. 
I'll see if I can fix this, since it's not the first time this happens. Thanks for making me check it again :)